include/boost/geometry/arithmetic/determinant.hpp

```diff
@@ -39,7 +39,13 @@ public :
     static inline ReturnType apply(U const& ux, U const& uy
                                  , V const& vx, V const& vy)
     {
-        return rt(ux) * rt(vy) - rt(uy) * rt(vx);
```

Comment:
Unfortunately, it gives the same result as the original. Btw, I'm compiling using MinGW-w64, it's also shipped with QtCreator for Windows, in case you'd like to play with it.